packages/manager/sms/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@ovh-ux/manager-sms",
```

Comment:
It would be great if you add some meta informations `description`, `keywords`:)

Reply:
done